app/src/main/java/com/monkeyteam/chimpagne/model/database/QueryFilters.kt

```diff
@@ -52,3 +52,7 @@ fun happensOnThisDateFilter(calendar: Calendar): Filter {
       Filter.and(startsAfterFilter(startTimestampValidity), endsBeforeFilter(endTimestampValidity)),
       Filter.and(startsBeforeFilter(endTimestampValidity), endsAfterFilter(endTimestampValidity)))
 }
+
+fun eventByOwnerFilter(ownerID: String): Filter {
```

Comment:
never used nor tested, is it useful ?

Reply:
Indeed, it was something I used in a previous implementation where I used to search the events of an owner directly in the database. 
Now it is useless so we can delete it.